RELEASING.md

```diff
@@ -12,3 +14,15 @@ These are the steps to release Gradle profiler.
 * Update the [release draft](https://github.com/gradle/gradle-profiler/releases) prepared by [release-drafter](https://probot.github.io/apps/release-drafter/), assign the tag you created and publish the release
 * The [Publish to SDKman](https://builds.gradle.org/buildConfiguration/GradleProfiler_GradleProfilerPublishToSdkMan?branch=%3Cdefault%3E) should trigger automatically after the publishing job finishes. Though it may fail, since the artifacts are not yet on Maven Central. The build will retry the SDKMan publishing after half an hour. If it still doesn't work, you can trigger the build manually. 
 * All done!
+
+### Releasing Snapshot Versions
+
+Releasing master snapshot versions is also supported.
+These versions are normally used for dogfooding in the [gradle/gradle](https://github.com/gradle/gradle) build.
+We use `-alpha` suffix for these versions, e.g. `0.20.0-alpha01`.
```

Comment:
❔ Can we use `-alpha-1` format instead? Would match the format we use elsewhere, like in Gradle's `8.0-milestone-1` etc.

Reply:
Sure why not, we can start with that format with next version